specifyweb/frontend/js_src/lib/components/Statistics/index.tsx

```diff
@@ -42,9 +42,15 @@ import { defaultLayoutGenerated, dynamicStatsSpec } from './StatsSpec';
 import type { CustomStat, DefaultStat, StatLayout } from './types';
 import { ProtectedAction } from '../Permissions/PermissionDenied';
 
-const TIME_DIFF_MINUTE = 60 * 24;
+export function StatsPage(): JSX.Element {
+  return (
+    <ProtectedAction resource={'/querybuilder/query'} action={'execute'}>
+      <ProtectedStatsPage />
+    </ProtectedAction>
+  );
+}
 
-export function StatsPage(): JSX.Element | null {
+function ProtectedStatsPage(): JSX.Element | null {
   // TODO: Make stats page component smaller
```

Comment:
can you please replace usages of `TODO:` with better alternatives?
see https://github.com/specify/specify7/tree/production/specifyweb/frontend/js_src/lib#todos